packages/strapi-design-system/src/DateTimePicker/__tests__/DateTimePicker.e2e.js

```diff
@@ -0,0 +1,137 @@
+const { injectAxe } = require('axe-playwright');
+
+const { test, expect } = require('@playwright/test');
+
+test.describe.parallel('DateTimePicker', () => {
+  test.describe('without initial data', () => {
+    test.beforeEach(async ({ page }) => {
+      // This is the URL of the Storybook Iframe
+      await page.goto('/iframe.html?id=design-system-components-datetimepicker--base&viewMode=story');
+      await injectAxe(page);
```

Comment:
Why are you injecting axe before each test ?

Reply:
And you are not doing any `checkA11y`, injecting axe is useless without `checkA11y`

Reply:
yes I forgot to remove it

Reply:
removed thanks for the comment